content/en/docs/concepts/workloads/pods/pod-qos.md

```diff
@@ -105,7 +105,7 @@ mechanisms that both provide controls over quality of service.
 
 Certain behavior is independent of the QoS class assigned by Kubernetes. For example:
 
-* Any Container exceeding a resource limit will be killed and restarted by the kubelet without
+* Any Container exceeding a memory limit will be killed and restarted by the kubelet without
   affecting other Containers in that Pod.
```

Comment:
I can see why you've proposed this change, but:
-
 actually it's often not the kubelet that kills the container
- for memory, OOM kill may not trigger the moment that the resource limit is exceeded
- Kubernetes might one day add support for another resource that has a similar allocation to memory, and the original text would be more helpful

[Containers in] Pods can never use more CPU than their limit, because the kernel intercedes. However, that's more a special quality of the CPU resource, where you can't exceed the limit even if you try to.

If someone made a mechanism for containers to be able to use more CPU (eg cooperative realtime), but with a risk of termination if the task didn't yield as agreed, the original statement might still apply.

I am not sure we should make this change as drafted.